README.md

```diff
@@ -166,6 +166,7 @@ In order to train a new classifier, you must provide:
    * We recommend filtering out entries with a very low probability: removing those with a probability 10 times lower than the maximum translation probability for each word speeds up the process and does not decrease accuracy.
    * Prior to inferring the probabilistic dictionaries, sentences must be tokenizer with the Moses tokenizer (with the `-a` flag) and lowercased.
    * You can uses Moses and MGIZA++ to obtain probabilistic dictionaries from a parallel corpus.
+* Gzipped lists of word frequencies for SL and TL.
```

Comment:
Plase explain this :)

Reply:
I'd add something like "... obtained from another corpus (i.e. newscrawl)", or similar.

Reply:
Updated readme